ultrasonic/src/main/kotlin/org/moire/ultrasonic/service/PlaybackStateSerializer.kt

```diff
@@ -43,8 +43,8 @@ class PlaybackStateSerializer : KoinComponent {
     ) {
         if (!setup.get()) return
 
-        ioScope.launch {
-            if (lock.tryLock()) {
+        if (lock.tryLock()) {
```

Comment:
This won't work, the unlock will be called from another thread which throws an exception.